packages/usa-accordion/src/styles/_usa-accordion.scss

```diff
@@ -52,7 +82,14 @@ $accordion-button-background-active-color: next-token(
   }
 
   @media (forced-colors: active) {
-    forced-color-adjust: none;
+    border: $border-high-contrast;
+    position: relative;
+
+    &::after {
```

Comment:
I noticed that this update results in similar issues with banner and nav that https://github.com/uswds/uswds/pull/5269 had, as detailed in [this comment](https://github.com/uswds/uswds/pull/5269#issuecomment-1563550778). It might be necessary to manually remove the icon from banner and header nav, similar to what was done in 95de5f9888425057bd5760e5ecb810fba6541b33.

Banner:
![image](https://github.com/uswds/uswds/assets/93996430/549d68f2-b00e-4081-94ff-ffefa85f1bc6)

![image](https://github.com/uswds/uswds/assets/93996430/2cfe63d3-70f2-4207-9e63-f3e2540a78af)

Header nav:
![image](https://github.com/uswds/uswds/assets/93996430/e6bd5135-8f12-4326-8d10-e3926936b94e)

![image](https://github.com/uswds/uswds/assets/93996430/b6d6919a-b8c6-4642-9bc2-817ad685fe7a)

Reply:
I was able to resolve this by changing the accordion high contrast icon from an `::after` pseudo element to `::before` and then setting the display to none on banner and nav components.

Do you see an issue with resolving it this way?

https://github.com/uswds/uswds/blob/4ed6bb72c76ac5feb84a511702fb545641c7f975/packages/usa-nav/src/styles/_usa-nav.scss#L212-L225

Reply:
I _think_ this sounds reasonable

Reply:
Using `:before` in place of `:after` here makes sense to me since it will minimize interaction with the `:after` pseudo elements on in banner and mobile nav.